### ISSUE TITLE
Ribbons representation not displaying for DNA

### ISSUE THREAD

**Issue by avirshup** [Autodesk/notebook-molecular-visualization#29]
To reproduce:

```
mol = mdt.build_dna_helix('ACTGCGTA')
viewer = mol.draw(display=True)
viewer.ribbon()
```
## got:

![image](https://cloud.githubusercontent.com/assets/9388007/19825917/ec21c054-9d34-11e6-8c6a-cf76aae1a3e2.png)
## Result with moldesign 0.7.2 / nbmolviz 0.6.5:

![image](https://cloud.githubusercontent.com/assets/9388007/19825931/2786a2f4-9d35-11e6-8430-a3ad08c399c4.png)

**Comment by justinmc** [comment]
@avirshup I might need to sit down with you for a minute tomorrow to look into this.  I'm not able to draw this directly in 3dmol.  Using the data from that molecule, then trying to set the style for the whole molecule to "cartoon" just results in a blank page (like you saw).

Do you remember if that's how you were originally drawing these?  With "cartoon"?  Anything else that might have gotten removed from the code?

**Comment by avirshup** [comment]
Yeah - "cartoon" sounds right. Happy to sit down with this; it might be helpful to try comparing the 3DMol JSON data between the working and non-working versions.

I remember having this same problem a _very long_ time ago when I first started working on the project, but I have no idea how it got fixed the first time. My guess would be that this has something to do with 3DMol not recognizing the molecular data specifically as DNA - maybe a naming issue or something?

**Comment by avirshup** [comment]
Actually, I remember how I fixed this the first time - it was by passing my molecules to 3DMol as PDB files. I think 3DMol's PDB parser must be inserting some piece of metadata into its JSON representation that allows it to recognize a chain as a piece of DNA.

**Comment by justinmc** [comment]
That was it!  I was able to render the cartoon dna via pdb.  It's a hard problem to get it to work with the rest of MDT though...  I'm going to try digging into the 3dmol source and see exactly why it doesn't work with json.  Ideally I can submit a PR to them to fix this, or as a fallback I can probably hack in the extra metadata in our code.
